lib/views/reporting/reporting_card.dart

```diff
@@ -241,16 +317,97 @@ class ReportingCardViewState extends State<ReportingCardView> {
               _updateDescription(_controller.text);
               widget.onPressed();
             },
-            child: Text(widget.buttonText),
+            child: Text(_descriptionNotEmpty ? "Volgende" : widget.buttonText),
```

Comment:
widget.buttonText.isNotEmpty ? widget.buttonText : "volgende"

Dit bespaart je een aantal onnodige regels code

Reply:
Op deze manier opdate de button text niet als je begint te typen in de text field.